src/controller/model/group.ts

```diff
@@ -358,7 +358,7 @@ export class Group extends Entity {
                 Zcl.FrameType.SPECIFIC,
                 optionsWithDefaults.direction,
                 true,
-                optionsWithDefaults.manufacturerCode,
+                cluster.manufacturerCode,
```

Comment:
should I maybe do `optionsWithDefaults.manufacturerCode ?? cluster.manufacterCode` instead, so that if something explicitly passes in a manufacturerCode, it doesn't get overwritten?